Lib/http/server.py

```diff
@@ -131,6 +132,7 @@
 """
 
 DEFAULT_ERROR_CONTENT_TYPE = "text/html;charset=utf-8"
+RANGE_REGEX_PATTERN = re.compile(r'bytes=(\d*)-(\d*)$', re.IGNORECASE | re.ASCII)
```

Comment:
```suggestion
RANGE_REGEX_PATTERN = re.compile(r'bytes=(\d*)-(\d*)$', re.ASCII | re.IGNORECASE)
```

Nit (I think it becomes more readable if flags are alphabetically sorted)